### PR TITLE
Test user config

### LINKED ISSUE #1540: Test user_config.dart
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file:
```
lib/services/user_config.dart
```
- When complete, all methods, classes and/or functions in the file will need to be tested. 
- Our `test/` folder has the same structure as the `lib` folder. Place your test file in the equivalent folder under `test/`. You may need to create the appropriate directory structure to do this.
- [Current test code coverage for this file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa?search=&displayType=list)

### IMPORTANT:
Please refer to this issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)

### PR DESCRIPTION
**Pull request by Azad99-9**
**What kind of change does this PR introduce?**

This PR adds test to user config.dart file.

Fixes #1540 

**Did you add tests for your changes?**

yes

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

Motivation is to make user_config.dart file to 100% code covered.

**Does this PR introduce a breaking change?**

No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

yes

### CODE REVIEW

**Comment by Azad99-9** [comment]
@noman2002 @Cioppolo14 @Kevoniat please assign reviewers for this PR.

File: test/service_tests/user_config_test.dart [hunk 1]
```diff
@@ -170,5 +245,31 @@ void main() async {
       expect(mockUser.authToken, newAuthToken);
       expect(mockUser.refreshToken, newRefreshToken);
     });
+
+    test('Test for saveCurrentOrgInHive method.', () async {
```
Comment (literalEval) [inline test/service_tests/user_config_test.dart:249]: How is this testing saveCurrentOrgInHive method ? Please explain.

File: test/service_tests/user_config_test.dart [hunk 2]
```diff
@@ -78,7 +79,81 @@ void main() async {
...
+            'lastName': 'Doe',
+            'email': 'johndoe@example.com',
+            'image': 'https://example.com/profile.jpg',
+            'accessToken': 'exampleAccessToken',
+            'refreshToken': 'exampleRefreshToken',
+          }
+        ],
+      };
+
+      when(
```
Comment (literalEval) [inline test/service_tests/user_config_test.dart:118]: These simulation `when`(s) can be called in `setupAll` or where the mock is defined.

**Review by literalEval — CHANGES_REQUESTED** [review]

File: test/service_tests/user_config_test.dart (hunk 1 above)
Comment (Azad99-9) [reply test/service_tests/user_config_test.dart:249]: The test calls the `saveCurrentOrgInHive` method, storing the `_currentOrg` object in the `currentOrg` Hive box. The test assumes that if there's any problem while saving orgInfo into the Hive box, the `box.put` method will throw an exception, causing the test to fail. A successful test execution confirms the successful storage of orgInfo within the currentOrg Hive box.

**Comment by Azad99-9** [comment]
@literalEval please review this PR.

**Review by noman2002 — APPROVED** [review]
It looks fine to me. @literalEval Please review.

File: test/service_tests/user_config_test.dart (hunk 1 above)
Comment (literalEval) [reply test/service_tests/user_config_test.dart:249]: Alright. Add a comment explaining this.

File: test/service_tests/user_config_test.dart (hunk 2 above)
Comment (literalEval) [reply test/service_tests/user_config_test.dart:118]: Alright

**Review by literalEval — APPROVED** [review]
LGTM. Add said comments.

**Pull request merged**